### ISSUE TITLE
Throwing Error When Loading a Font

### ISSUE THREAD

**Issue by azamsharp** [material-foundation/google-fonts-flutter#8]
The google_fonts package is throwing errors when used. 

```
import 'package:flutter/material.dart';
import 'package:google_fonts/google_fonts.dart';

void main() => runApp(App());

class App extends StatelessWidget {

  @override
  Widget build(BuildContext context) {

    return MaterialApp(
      home: Scaffold(
        appBar: AppBar(
          title: Text("Google Fonts")
        ),
        body: Text("Google Fonts", style: GoogleFonts.lato())
      )
    );
    
  }

}
```

Exception has occurred.
MissingPluginException (MissingPluginException(No implementation found for method getApplicationDocumentsDirectory on channel plugins.flutter.io/path_provider))

**Comment by clocksmith** [comment]
Does this still happen after running `flutter pub get`?

**Comment by tirth-aubergine** [comment]
Try adding `WidgetsFlutterBinding.ensureInitialized();` in `void main` before the `runApp` call.

**Comment by l2D** [comment]
> I used 0.2.0 and had the same issue. So, I step back from 0.2.0 to 0.1.1 and not found an error.

### In pubspec.yaml
**dependencies**:
...
`google_fonts: 0.1.1`

And save, Flutter will get the package automatically.

**Comment by johnsonmh** [comment]
Has this been resolved? My hunch is the same as @clocksmith, seems like a `pub` issue that should be resolved by running `pub get`.

**Comment by WhiteNazguls** [comment]
Try to restart the running app. It's worked for me..

**Comment by maheshmnj** [comment]
> Try to restart the running app. It's worked for me..

this worked for me thanks

**Comment by SwaroopNandy2109** [comment]
> Try to restart the running app. It's worked for me..

Worked for me as well 👍

**Comment by sberney** [comment]
Ever since I started working on my flutter project again and upgraded from version `^0.3.2` to `^1.1.0`, every time I start the application, I get the same error message. The debugger in vscode breaks on the error, and I have to hit F5 to continue a lot.

I suspect that the issue is related to not having an offline version of the fonts, which I am not interested in correcting. I want to dynamically download fonts so I can tweak the styles in my app and find the best one for the situation. When it comes time to distribute the app, I'll want to download the offline files for whatever I've ended up picking.

I've tried all the suggestions I can find, and pretty much just hit F5 a lot (10-15 time) every time I reload or stop and start the application.
